boards/microbit_v2/Makefile

```
@@ -21,11 +20,11 @@ install: flash
 
 .PHONY: flash-debug
 flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<;  reset; shutdown;"
+	$(PROBE_RS) download --chip nRF52833_xxAA --verify $<	
```

i.e., it seems like this should be something more like
```suggestion
ifdef USE_PROBE_RS
	$(PROBE_RS) download --chip nRF52833_xxAA --verify $<	
else
        $(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<; reset; shutdown;"
endif
```

, at least until all the openocd stuff could reasonably be removed from the board. Promoting a half-complete thing as the default isn't a great end user experience I don't think.

---

made the necessary changes, but i feel like the option to use probe-rs is not visible enough in the terminal